src/inputs/labeled-inputs.scss

```diff
@@ -212,13 +225,13 @@
     }
```

Comment:
need to check these paddings.. and selector. Maybe we dont need to duplicate native textarea and input? would be enough our classes..

Reply:
Selector changed